calculatedmetrics.md

```diff
@@ -1,10 +1,287 @@
+
+```
+curl -X GET “https://analytics.adobe.io/api/[company name]/calculatedmetrics?locale=en_US&limit=10&page=0” -H “x-api-key: [oauth token]” -H “x-proxy-global-company-id: [company name]” -H “Authorization: Bearer [ims user token]” -H “Accept: application/json” -H “Content-Type: application/json”
 ```
-curl -X GET "https://analytics.adobe.io/api/{COMPANY ID}/calculatedmetrics?rsids={RSID}&locale=en_US&limit=10&page=0" -H  "x-api-key: {API KEY}" -H  "x-proxy-global-company-id: {COMPANY ID}" -H  "Authorization: {BEARER TOKEN}  -H  "Accept: application/json" -H  "Content-Type: application/json"
+
+## Example response
+
+The following example shows the response for the previous request, including the predefined calculated metrics for Bounce Rate and Revenue per Visitor (`cm_bouncerate_defaultmetric` and `cm_revenue_visitor_defaultmetric`):
+
 ```
+[    
+    {
+      "id": "cm_bouncerate_defaultmetric",
+      "name": "Bounce Rate",
+      "description": "Default Bounce Rate Metric",
+      "polarity": "positive",
+      "precision": 1,
+      "type": "percent",
+      "definition": {
+        "formula": {
+          "col": {
+            "func": "divide",
+            "col2": {
+              "func": "metric",
+              "name": "metrics/entries",
+              "description": "Entries"
+            },
+            "col1": {
+              "func": "metric",
+              "name": "metrics/bounces",
+              "description": "Bounces"
+            }
+          },
+          "func": "visualization-group"
+        },
+        "func": "calc-metric",
+        "version": [
+          1,
+          0,
+          0
+        ]
+      },
+      "template": true,
+      "categories": [
+        "Calculated Metrics"
+      ]
+    },
+    {
+      "id": "cm_revenue_visitor_defaultmetric",
+      "name": "Revenue / Visitor",
+      "description": "Default Revenue / Visitor Metric",
+      "polarity": "positive",
+      "precision": 2,
+      "type": "currency",
+      "definition": {
+        "formula": {
+          "func": "divide",
+          "col2": {
+            "func": "metric",
+            "name": "metrics/visitors"
+          },
+          "col1": {
+            "func": "metric",
+            "name": "metrics/revenue"
+          }
+        },
+        "func": "calc-metric",
+        "version": [
+          1,
+          0,
+          0
+        ]
+      },
+      "template": true,
+      "categories": [
+        "Calculated Metrics"
+      ]
+    }
+```
+
+## Retrieving a single calculated metric
+
+To retrieve a single calculated metric, include its `id` in the request.
+
+### Example request
+
+```
+curl -X GET “https://analytics.adobe.io/api/[company name]/calculatedmetrics/[calculated metrics id]?locale=en_US” -H “x-api-key: [oauth token]” -H “x-proxy-global-company-id: [company name]” -H “Authorization: Bearer [ims user token]” -H “Accept: application/json” -H “Content-Type: application/json”
```

Comment:
The path should have `[calculated metric id]` (singular). Also, I need to double check if the `x-proxy-global-company-id` expects a companyId or the companyName. That header name may be incorrect.